tests/networkpolicy_test.go

```diff
@@ -176,19 +238,26 @@ func skipNetworkPolicyRunningOnKindInfra() {
 	}
 }
 
-func createVMICirros(virtClient kubecli.KubevirtClient, namespace string, labels map[string]string) *v1.VirtualMachineInstance {
+func createVMICirros(virtClient kubecli.KubevirtClient, namespace string, labels map[string]string, opts ...libvmi.Option) *v1.VirtualMachineInstance {
 	var err error
 	vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
 	vmi.Namespace = namespace
 	vmi.Labels = labels
+
+	// Clean up interfaces since we configure them with `libvmi.Option`
+	vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{}
```

Comment:
do we want to hint about this intrusive change, in the function name, in case someone use it and doesnt notice?
(i am not sure we do, but raising the question)

Reply:
I do not understand your comment here. What intrusive change caused here? All of this is internal to the function.